acslib/ccure/crud.py

```diff
@@ -62,14 +75,60 @@ def count(self) -> int:
             ),
         ).json
 
-    def update(self, record_id: str, update_data: dict) -> ACSRequestResponse:
-        pass
+    def update(self, personnel_id: int, update_data: dict) -> ACSRequestResponse:
+        """
+        Edit properties of a personnel object
 
-    def create(self, create_data: dict) -> ACSRequestResponse:
-        pass
+        :param personnel_id: the Personnel object's CCure ID
+        :param update_data: maps Personnel properties to their new values
+        """
+        return self.connection.request(
+            ACSRequestMethod.PUT,
+            request_data=ACSRequestData(
+                url=self.config.base_url + self.config.endpoints.EDIT_OBJECT,
```

Comment:
PUT to CREATE an object? Shaking my Damn Head. Still the most bonkers API I've ever seen.